src/app/client/src/app/modules/public/module/explore/components/explore-content/explore-content.component.ts

```diff
@@ -223,14 +225,20 @@ export class ExploreContentComponent implements OnInit, OnDestroy {
             queryParams: this.queryParams
         });
     }
-
+    getFilters(filters) {
+        _.forEach(filters, (value) => {
+            if (value.code === 'board') {
+               this.dataDrivenFilter = _.get(value, 'range[0].name') ? _.get(value, 'range[0].name') : [];
```

Comment:
this.dataDrivenFilter. board = _.get(value, 'range[0].name') ? _.get(value, 'range[0].name') : [];